### DISCUSSION TITLE
How to handle internationalized languages during vscode source code development?

### DISCUSSION THREAD

**Discussion by yiliang114** [microsoft/vscode-discussions#732] [Extension Development QnA] [answered]
In the development phase, `vs/nls` doesn't seem to handle internationalized language plug-ins. 

That is to say, now debug vscode source code, can not choose a language other than English. For example, I am Chinese. If I am debugging vscode source code and it shows Chinese, it will be more friendly to me.

Does anyone have a solution?

**Comment by yiliang114** [comment]
Hi @Tyriar is there any way?

**Reply by Tyriar** [reply]
Please don't ping random people, the dev side generally doesn't get involved in vscode-discussions as we already have our hands full with incoming issues.

**Comment by digitarald** [comment, marked as answer]
I'd recommend filing an issue in https://github.com/microsoft/vscode-l10n . The use case surely makes sense, and testing extensions in different locales is important.
